pymc4/distributions/discrete.py

```diff
@@ -176,11 +177,13 @@ class BetaBinomial(BoundedDiscreteDistribution):
     """
 
     def __init__(self, name, total_count, concentration0, concentration1, **kwargs):
+        concentration0 = np.array(concentration0)
```

Comment:
Don't try to convert to numpy arrays. You should leave them alone or `tf.convert_to_tensor`